content/en/docs/setup/production-environment/turnkey-cloud-solutions.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Turnkey Cloud Solutions
+content_type: concept
+weight: 50
+reviewers:
+# alibaba
+- colemickens
+- brendandburns
+# aws
+- justinsb
+- clove
+# azure
+- colemickens
+- brendandburns
+# gce
+- brendandburns
+- jbeda
+- mikedanese
+- thockin
+# ibm
+- bradtopol
+---
+
+<!-- overview -->
+
+This page contains a list of production ready Kubernetes cluster providers.
```

Comment:
Rather than listing them, we could:
- mention that providers do offer “turn key” solutions
- use the CNCF landscape to list recognised offerings. See https://github.com/kubernetes/website/pull/24014 for related work to make the existing shortcode more generic.

Reply:
You would rather present "production ready" solutions on a single page instead
of removing the content?

Reply:
Once #24104 merges we can use that shortcode to revise https://kubernetes.io/docs/setup/production-environment/turnkey/

- explain that turnkey solutions exist
- outline the certification process, any other context we want to tell readers about
- embed https://landscape.cncf.io/category=certified-kubernetes-distribution,certified-kubernetes-hosted&format=logo-mode&grouping=category&embed=yes

Reply:
I am not convinced that this content needs to exist or explain to readers about "turnkey" solutions (more appropriate for sales/marketing?).
I am leaning toward removing the page and let readers peruse the "landscape" page on their own.